packages/components/src/__actions__/Button/v3/_docs/Button--usage-guidelines.mdx

```diff
@@ -0,0 +1,32 @@
+import { Canvas, Meta, Controls } from "@storybook/blocks"
+import { ResourceLinks, Installation } from "~storybook/components"
+import * as Button from "./Button.stories"
+
+<Meta title="Actions/Button/Button (v3)/Usage Guidelines" />
+
+# Button (v3)
+
+Updated July 12, 2024
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__actions__/Button/v3"
+  figma="https://www.figma.com/design/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-Heart-UI-Kit?m=auto&t=a6VcoKBJx6snn0ut-1"
+  ariaComponentPage="https://react-spectrum.adobe.com/react-aria/Menu.html"
```

Comment:
What's this? Doesn't seem to show up

Reply:
Oh yeah, that's from way back when that I ended up removing from the ResourceLinks component, tis now gone 👋